mutiny-wasm/src/lib.rs

```diff
@@ -1126,6 +1126,31 @@ impl MutinyWallet {
             .into())
     }
 
+    /// Create a budgeted nostr wallet connect profile
+    #[wasm_bindgen]
+    pub async fn create_budget_nwc_profile(
```

Comment:
Is there already a method for editing these? changing the budget / max use for a given profile?

Reply:
Yeah, might be a bit difficult for the front end to use, maybe can add a set budget option

Reply:
Still think we need this.